### PR TITLE
Ahodges/a11y style guide

### PR DESCRIPTION
**Pull request by lamagnifica**
@mhoeber @srpearce @catong I added some info collected by DOC-1960 for accessibility issues.
@cptvitamin FYI

### CODE REVIEW

File: en_us/edx_style_guide/source/images.rst [hunk 1]
```diff
@@ -120,7 +129,41 @@ When you add an image to a file, include three lines.
     :alt: An empty HTML component editor in Studio
 
 
+*****************
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/images.rst]: @cptvitamin here's the new section on alt text for images

File: en_us/edx_style_guide/source/preferred.rst [hunk 2]
```diff
@@ -69,6 +68,11 @@ Preferred Usage
       checklists<Use the Course Checklist>``` to verify that the course is
       ready for release."
```
Comment (lamagnifica) [inline en_us/edx_style_guide/source/preferred.rst:70]: @cptvitamin new info on accessible cross references

File: en_us/edx_style_guide/source/images.rst [hunk 3]
```diff
@@ -120,7 +129,41 @@ When you add an image to a file, include three lines.
...
+
+.. code-block:: xml
+
+ :alt: Image of a multiple choice problem.
+
+ :alt: Example response.
+
+When you write alt text, follow these guidelines. 
+
+* Be aware of how long the alt text is. Automated tests produce warnings for
```
Comment (cptvitamin) [inline en_us/edx_style_guide/source/images.rst]: @lamagnifica there is no limit to the length of alt text. Some automated tests will fail long alt text, but there is no limit defined anywhere.  There are practical limits.  You don't want it to be too long, but at the same time, I don't want authors to worry about being so concise that the quality suffers.

File: en_us/edx_style_guide/source/preferred.rst [hunk 4]
```diff
@@ -69,6 +68,11 @@ Preferred Usage
...
 
+      To promote a better experience for screenreaders, provide the title of
+      the destination in each cross-reference link, and not just a URL. In
+      addition, avoid including multiple links to the same destination on a
+      single HTML page.
```
Comment (mhoeber) [inline en_us/edx_style_guide/source/preferred.rst:74]: ^^ we likely need a good bit of work to adhere to this.

File: en_us/edx_style_guide/source/images.rst [hunk 5]
```diff
@@ -120,7 +129,41 @@ When you add an image to a file, include three lines.
     :alt: An empty HTML component editor in Studio
 
 
+*****************
+Accessibility
+*****************
+
+Every image added to the documentation must have alt text that makes the
+purpose of the image clear to those who are using screen readers.
```
Comment (cptvitamin) [inline en_us/edx_style_guide/source/images.rst:138]: Can we add that "alt text should serve as a functional equivalent for the image." or something of that nature?

**Comment by mhoeber** [comment]
looks good.  :+1:

**Comment by mhoeber** [comment]
build error is in this guide

File: en_us/edx_style_guide/source/images.rst (hunk 5 above)
Comment (lamagnifica) [inline en_us/edx_style_guide/source/images.rst:138]: added as initial sentence!

File: en_us/edx_style_guide/source/images.rst (hunk 3 above)
Comment (lamagnifica) [inline en_us/edx_style_guide/source/images.rst]: reworded to make the priorities clearer

**Comment by lamagnifica** [comment]
fixed the build error

**Comment by lamagnifica** [comment]
@cptvitamin, some rewrites included per your comments. @srpearce and @catong just FYI.

**Pull request merged**